print-ldp.c

```diff
@@ -328,11 +328,17 @@ ldp_tlv_print(netdissect_options *ndo,
 	break;
 
     case LDP_TLV_COMMON_SESSION:
-	TLV_TCHECK(8);
+	TLV_TCHECK(14);
 	ND_PRINT("\n\t      Version: %u, Keepalive: %us, Flags: [Downstream %s, Loop Detection %s]",
 	       GET_BE_U_2(tptr), GET_BE_U_2(tptr + 2),
-	       (GET_BE_U_2(tptr + 6)&0x8000) ? "On Demand" : "Unsolicited",
-	       (GET_BE_U_2(tptr + 6)&0x4000) ? "Enabled" : "Disabled"
+	       (GET_BE_U_2(tptr + 4)&0x8000) ? "On Demand" : "Unsolicited",
+	       (GET_BE_U_2(tptr + 4)&0x4000) ? "Enabled" : "Disabled"
+	       );
+	ND_PRINT("\n\t      Path Vector Limit %u, Max-PDU length: %u, Receiver Label-Space-ID %s:%u",
+	       *(tptr+5),
```

Comment:
Thanks for the PR.
There should be a GET_*() here unstead of direct dereferencing.
Also: Do you have any PCAPs showing these changes?

Reply:
- have replaced the direct reference with a GET_U_1()
- I have a pcap but cannot attach it in github.
- output looks like this ```12:24:11.046149 IP (tos 0xc0, ttl 255, id 1506, offset 0, flags [none], proto TCP (6), length 81)
    192.168.0.2.58321 > 192.168.0.1.646: Flags [P.], cksum 0x68c1 (correct), seq 110237:110278, ack 87765, win 3025, length 41
        LDP, Label-Space-ID: 192.168.0.2:0, pdu-length: 37
          Initialization Message (0x0200), length: 27, Message ID: 0x00000001, Flags: [ignore if unknown]
            Common Session Parameters TLV (0x0500), length: 14, Flags: [ignore and don't forward if unknown]
              Version: 1, Keepalive: 30s, Flags: [Downstream Unsolicited, Loop Detection Enabled]
              Path Vector Limit 32, Max-PDU length: 0, Receiver Label-Space-ID 192.168.0.1:0
            Unknown TLV (0x050b), length: 1, Flags: [continue processing and don't forward if unknown]
              0x0000:  80```

Reply:
> * I have a pcap but cannot attach it in github.

Could you add it as a new test in the tests/ directory?

Reply:
done !